tools/convert/README.md

```diff
@@ -555,19 +705,19 @@ gfxrecon-convert --output stdout vkcube.f1.gfxr | egrep -v "^{\"header\":{\"" |
 This can be stripped-down to just the values:
 
 ```bash
-gfxrecon-convert --output stdout vkcube.f1.gfxr | egrep -v "^{\"header\":{\"" | jq -c '.vkFunc.args = (.vkFunc.args | to_entries | map_values(.value))'
+gfxrecon-convert --output stdout vkcube.f1.gfxr | egrep -v "^{\"header\":{\"" | jq -c '.function.args = (.function.args | to_entries | map_values(.value))'
```

Comment:
I think this example and other examples may need to be updated to add `--format jsonl` when appropriate